CHANGELOG.md

```diff
@@ -3,3 +3,9 @@
 ## v0.4.1-beta
 
 Initial public release
+
+## Unreleased changes
+
+### Breaking changes
+
+- Added new abstract property `TaskOrchestrationContext.ParentInstance`.

```

Comment:
The `v0.4.1` release already went out, so let's add to the top of this file a new `v0.5.0-beta` section and put your new content there. Here's how I've been doing it for the MSSQL backend for DTFx: https://github.com/microsoft/durabletask-mssql/blob/main/CHANGELOG.md (I originally copied the approach of the [KEDA project's CHANGELOG.md file](https://github.com/kedacore/keda/blob/main/CHANGELOG.md)).